react/week4/src/components/UsersFetch.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { UserProvider } from './UserContext';
+import UsersSearch from './UsersSearch';
+import UsersList from './UsersList';
+
+export default function UsersFetch() {
+    const [users, setUsers] = useState("")
+    const [result, setResult] = useState([{ id: 1, login: "No result" }]);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState("");
+
+    useEffect(() => {
+        async function fetchApi(query) {
+            if (query.length === 0) {
+                setResult(() => [{ id: 1, login: "No result" }])
```

Comment:
Instead of setting result here, I think it is better to check if result length is bigger than 0 in the UsersList component and then show the list, or if it is an empty array show No result text